client/src/App.tsx

```diff
@@ -20,11 +21,17 @@ function App(): JSX.Element {
           <AuthProvider>
             <SocketProvider>
               <Switch>
-                <Route exact path="/login" component={Login} />
-                <Route exact path="/signup" component={Signup} />
-                <Route exact path="/booking" component={Booking} />
+                <Route exact path="/login">
+                  <Layout component={<Login />} />
```

Comment:
I think we can create Layout as a wrapper component and use within each individual component here - so that we can have cleaner routes

Reply:
Please give me examples. I cannot catch your idea.

Reply:
I've changed the above comment to optional but for an example, please see below:

```
export default function Login() {

return(
<Layout>
<LoginForm />
........
</Layout>

)
}

```